yarn.lock

```diff
@@ -37967,7 +37938,6 @@ __metadata:
   version: 3.11.174
   resolution: "pdfjs-dist@npm:3.11.174"
   dependencies:
-    canvas: ^2.11.2
```

Comment:
This change, and a change later in `yarn.lock`, should prevent Yarn from installing `canvas` as a transitive dependency of `pdfjs-dist`. An alternative to modifying `yarn.lock` directly might be to use Yarn's `resolutions` to resolve `canvas@npm:^2.11.2` to a new `packages/empty` directory.